src/params.h

```diff
@@ -102,10 +106,14 @@ General options: \n\
   -iwad <file>   Specify an IWAD file to use.\n\
 \n\
 Game start options: \n\
+  -nm                 Alias for -skill 5.\n\
+  -nomo               Alias to -nomonsters.\n\
```

Comment:
"Alias for" or "Alias to"?

Reply:
I changed it to "Alias to" in https://github.com/fabiangreffrath/woof/pull/1678/commits/1afd91afb28b1c50ea5f9b0402c45f4443eae3ab